Hackney.Shared.HousingSearch/Gateways/Models/Accounts/QueryableAccount.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Hackney.Shared.HousingSearch.Domain.Accounts;
+using Hackney.Shared.HousingSearch.Domain.Accounts.Enum;
+using Nest;
+
+namespace Hackney.Shared.HousingSearch.Gateways.Models.Accounts
+{
+    public class QueryableAccount
+    {
+
+        public Account Create()
```

Comment:
Perhaps name this `ToAccount()` since it's not really creating a `QueryableAccount`?